woodwork/indexers.py

```diff
@@ -86,10 +86,30 @@ def _process_selection(selection, original_data):
         if schema:
             del schema['dtype']
             selection.ww.init(**schema, use_standard_tags=original_data.ww.use_standard_tags)
-    elif isinstance(selection, pd.DataFrame) or (ks and isinstance(selection, ks.DataFrame)):
+    elif _is_dataframe(selection):
         # Selecting a new DataFrame from an existing DataFrame
         schema = original_data.ww.schema
         new_schema = schema._get_subset_schema(list(selection.columns))
         selection.ww.init(schema=new_schema)
     # Selecting a single value or return selection from above
     return selection
+
+
+def _is_series(data):
```

Comment:
nifty! This might be useful in the column_accessor's `_make_series_call` (and the dataframe one eventually with the `_make_dataframe_call` in the table accessor)

Reply:
The `if` conditions in `_process_selection` were getting really messy without this. Seems like it is more readable this way.

Reply:
And, yeah ,we could add this to utils and use it elsewhere if that makes sense.

Reply:
Moved these to accessor_utils and updated _make_series_call to use _is_series.